schedule.md

```diff
@@ -0,0 +1,89 @@
+# Session plan
+
+## Session 1: R and statistical concepts used
+
+Some of this could be pre-course reading material, see e.g. https://sbfnk.github.io/mfiidd/slides/rintro.pdf
+
+Monday 9:30-11:00
+
+- Introduction to R concepts used in the course (15 mins)
+- Introduction to statistical concepts used in the course and what they look like in stan (30 mins)
+- Practice session: probability distribution and estimates (30 mins)
+- Wrapup (15 mins)
+
+## Session 2: Delay distributions
+
+Monday 11:00-12:30
+
+- Introduction to epidemiological delays and how to represent them with probability distributions (15 mins)
+- Practice session: simulate and estimate epidemiological delays (30 mins)
+- Practice session: introducing daily censoring (30 mins)
+- Wrapup (15 mins)
+
+## Session 3a: Delay distributions continued
+
+Monday 14:00-14:45
+
+- Practice session: truncation (45 mins)
+
+## Session 3b: Nowcasting
+
+- Introduction to nowcasting as a particular right-truncation problem (30 mins)
+- Practice session: nowcasting in R (45 mins)
+- Wrapup (15 mins)
+
+## Session 4: R estimation and the renewal equation
+
+Tuesday 09:00-10:30
+
+- Introduction to the time-varying reproduction number (20 mins)
+- Practice session: using the renewal equation to estimate R (45 mins)
+- Practice session: combining nowcasting and R estimation (25 mins)
+
+## Session 5: Forecasting concepts
+
+Tuesday 11:00-12:30
+
+- Wrap-up of previous session (15 mins)
+- Introduction to forecasting as an epidemiological problem, and its relationship with nowcasting and R estimation (10 mins)
+- Introduction to forecast evaluation (20 mins)
+- Practice session: evaluate the nowcasts from session 3b (45 mins)
```

Comment:
before we talk about eval can we give people 15 minutes to play with the model we will eval to make some forecasts (but not use these in the eval session?)

Reply:
I went back and forth on this. Is it worth first introducing what we're aiming for in forecast or first play with the model - very open to alternative approaches here.

Reply:
Yeah I'm really not sure. I quite like the idea of using the play as a jumping off point for eval (i.e so how do we know which of you to trust giving advice to policy makers etc etc) but I think either could work. 

We could also have the play be basically what is good or bad about these forecasts you are making (which they will struggle to answer) and then present the eval as that answer.